### ISSUE TITLE
1. How to get the lane details of ego vehicle and NPC vehicle using Python APIs, Lane details include the Lane ID, lane geometry ? 

### ISSUE THREAD

**Issue by Revbr** [lgsvl/simulator#1094]

**Comment by EricBoiseLGSVL** [comment]
@Revbr We are working on a sensor to output some lane data.  It should release soon.  In the meantime you can create a custom sensor that checks map annotation to publish this data.  You will need to do the calculations for 2.  Again you can put this in a custom sensor plugin.

**Comment by DeepAnonymous** [comment]
hi @EricBoiseLGSVL ,

I am wondering is "a sensor to output some lane data" supported now? Or is it any more in-depth guide to check map annotation and calculate lane details?

Thank you.

**Comment by EricBoiseLGSVL** [comment]
@DeepAnonymous Yes, we are working on a new public release out soon.  We hope to be ready by end of Jan.
